src/luascript.cpp

```diff
@@ -11957,6 +11976,18 @@ int LuaScriptInterface::luaVocationGetPromotion(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaVocationGetFromVocation(lua_State* L)
+{
+	// vocation:getFromVocation()
```

Comment:
`getDemotion` works in the same way. Why have two methods that do the same thing? Maybe it's better to add this other name to the same method?

`core/vocation.lua`
`Vocation.getFromVocation = Vocation.getDemotion`